mackup/applications/raycast.cfg

```diff
@@ -3,3 +3,7 @@ name = Raycast
 
 [configuration_files]
 Library/Preferences/com.raycast.macos.plist
+Library/Application Support/com.raycast.macos
```

Comment:
When I tried this, I got an error about broken database on a computer which I executed `mackup restore`. It seems we should not mackup sqlite database under Application Support directory.
<img width="862" alt="image" src="https://user-images.githubusercontent.com/37664775/207752763-75b82a88-0a39-4012-bb18-8bfaec9db7a6.png">

Reply:
There's additional steps here—you need to pull over the Raycast keychain entries. I can document this in the config file  if you'd like. Or if this isn't a good fit for mackup, let me know.